examples/notebooks/responsibleaidashboard-programmer-regression-model-debugging.ipynb

```diff
@@ -762,7 +762,7 @@
     "            insight_3=counterfactual_job.outputs.counterfactual,\n",
     "            insight_4=erroranalysis_job.outputs.error_analysis,\n",
     "        )\n",
-    "        rai_gather_job.set_limits(timeout=480)\n",
+    "        rai_gather_job.set_limits(timeout=7200)\n",
```

Comment:
are these synced with the changes in azureml-examples repository?  We increased the timeouts there already I thought.

Reply:
Yes! I copied from there. I almost wasn't able to decode the error message, but vaguely remembered the fix there and that's exactly what it was :-)